lib/ring_logger/client.ex

```diff
@@ -97,12 +99,21 @@ defmodule RingLogger.Client do
   end
 
   def init(config) do
+    server_env = Application.get_env(:logger, RingLogger, [])
+    client_env = Application.get_env(:logger, __MODULE__, [])
```

Comment:
The `server_env`/`client_env` lines aren't needed to support `:location`. Could you move those to a separate PR?